Source/Layout/ASLayout.mm

```diff
@@ -55,22 +55,6 @@ ASDISPLAYNODE_INLINE AS_WARN_UNUSED_RESULT BOOL ASLayoutIsDisplayNodeType(ASLayo
   return layout.type == ASLayoutElementTypeDisplayNode;
 }
 
-ASDISPLAYNODE_INLINE AS_WARN_UNUSED_RESULT BOOL ASLayoutIsFlattened(ASLayout *layout)
-{
-  // A layout is flattened if its position is null, and all of its sublayouts are of type displaynode with no sublayouts.
-  if (! ASPointIsNull(layout.position)) {
-    return NO;
-  }
-  
-  for (ASLayout *sublayout in layout.sublayouts) {
-    if (ASLayoutIsDisplayNodeType(sublayout) == NO || sublayout.sublayouts.count > 0) {
-      return NO;
-    }
-  }
-  
-  return YES;
-}
-
```

Comment:
Had to make this a method so that it can see the `_sublayouts` ivar for direct access (remove retain/release pair per sublayout).